src/server.ts

```diff
@@ -17,7 +17,74 @@ app.use(express.json()); //add JSON body parser to each following route handler
 app.use(cors()); //add CORS support to each following route handler
 
 app.get("/", async (_req, res) => {
-    res.json({ msg: "Hello! There's nothing interesting for GET /" });
+    res.json({
+        msg: "Hello! There's nothing here to GET/ Check out emdpoint /pastes",
+    });
+});
+
+app.get("/pastes", async (_req, res) => {
+    try {
+        const text =
+            "SELECT id,title, LEFT(body,50)AS body FROM pasteBin ORDER BY (id) DESC LIMIT 10";
```

Comment:
Consider if you need to grab only 50 characters, because it's not enough to show 5 lines summary. The alternative might be to get full body, but limit summary view by css in front-end.